packages/app-frontend/src/features/components/ComponentsInspector.vue

```diff
@@ -59,7 +59,7 @@ export default defineComponent({
     // Refresh
 
     function refresh () {
```

Comment:
This function is called when clicking on `Force refresh` button in the Vue devtools. Not sure this is related to the issue you describe. Could you open a new GitHub issue with a reproduction so we can discuss it and try fixing it?

Reply:
All right. I will reconfirm this code. if it can be reproduced I will create a issue . Thank you for your review.